plugins/package-managers/maven/src/main/kotlin/Tycho.kt

```diff
@@ -277,7 +277,7 @@ class Tycho(
         buildList {
             // The "package" goal is required; otherwise the Tycho extension is not activated.
             add("package")
-            add("dependency:tree")
+            add("org.apache.maven.plugins:maven-dependency-plugin:3.8.1:tree")
```

Comment:
Please add the gist of the commit message also as a code comment. And maybe extract the version string to a constant, and ad the comment only to that constant.

Reply:
Extracted a constant and added a corresponding comment.

Reply:
Thanks.

BTW @oheger-bosch, I wanted to bring https://github.com/oss-review-toolkit/ort/issues/4720 to your attention, specifically the observation that the `maven-dependency-plugin` [de-duplicates dependencies](https://github.com/oss-review-toolkit/ort/issues/4720#issuecomment-1110610163) unless run in [verbose](https://maven.apache.org/plugins/maven-dependency-plugin/tree-mojo.html#verbose) mode. So we should probably do that.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/9978.